src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -424,6 +424,33 @@ public virtual JsonResult Json(object data, [NotNull] JsonSerializerSettings ser
         }
 
         /// <summary>
+        /// Creates a <see cref="HttpOkResult"/> object that produces an empty OK (200) response.
+        /// </summary>
+        /// <returns>The created <see cref="HttpOkResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkResult Ok()
```

Comment:
That would be `Ok`. And also :ok: with me. I realize that we've got a hodge-podge here and can't really fix it for compat reasons.